src/Middleware/HttpsPolicy/test/HstsMiddlewareTests.cs

```diff
@@ -15,13 +15,152 @@
 using Microsoft.Extensions.Hosting;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Testing;
+using Microsoft.Extensions.Options;
 using Microsoft.Net.Http.Headers;
 using Xunit;
 
 namespace Microsoft.AspNetCore.HttpsPolicy.Tests
 {
     public class HstsMiddlewareTests
     {
+#region Unit tests
```

Comment:
No regions please. If the test are meaningfully different then they should go in different files.

Reply:
Removed